spec/controllers/likes_controller_spec.rb

```diff
@@ -1,146 +1,131 @@
-
-      describe '#create' do
-        let(:like_hash) {
-          {:positive => 1,
-           id_field => "#{@target.id}"}
-        }
-        let(:dislike_hash) {
-          {:positive => 0,
-           id_field => "#{@target.id}"}
-        }
-
-        context "on my own post" do
-          it 'succeeds' do
-            @target = alice.post :status_message, :text => "AWESOME", :to => @alices_aspect.id
-            @target = alice.comment!(@target, "hey") if class_const == Comment
-            post :create, like_hash.merge(:format => :json)
-            expect(response.code).to eq('201')
-          end
-        end
-
-        context "on a post from a contact" do
-          before do
-            @target = bob.post(:status_message, :text => "AWESOME", :to => @bobs_aspect.id)
-            @target = bob.comment!(@target, "hey") if class_const == Comment
-          end
-
-          it 'likes' do
-            post :create, like_hash
-            expect(response.code).to eq('201')
-          end
-
-          it 'dislikes' do
-            post :create, dislike_hash
-            expect(response.code).to eq('201')
-          end
-
-          it "doesn't post multiple times" do
-            alice.like!(@target)
-            post :create, dislike_hash
-            expect(response.code).to eq('422')
-          end
-        end
-
-        context "on a post from a stranger" do
-          before do
-            @target = eve.post :status_message, :text => "AWESOME", :to => eve.aspects.first.id
-            @target = eve.comment!(@target, "hey") if class_const == Comment
-          end
-
-          it "doesn't post" do
-            expect(alice).not_to receive(:like!)
-            post :create, like_hash
-            expect(response.code).to eq('422')
-          end
-        end
-
-        context "when an the exception is raised" do
-          before do
-            @target = alice.post :status_message, :text => "AWESOME", :to => @alices_aspect.id
-            @target = alice.comment!(@target, "hey") if class_const == Comment
-          end
-
-          it "should be catched when it means that the target is not found" do
-            params = like_hash.merge(format: :json, id_field => -1)
-            post :create, params
-            expect(response.code).to eq('422')
-          end
-
-          it "should not be catched when it is unexpected" do
-            @target = alice.post :status_message, :text => "AWESOME", :to => @alices_aspect.id
-            @target = alice.comment!(@target, "hey") if class_const == Comment
-            allow(alice).to receive(:like!).and_raise("something")
-            allow(@controller).to receive(:current_user).and_return(alice)
-            expect { post :create, like_hash.merge(:format => :json) }.to raise_error("something")
-          end
-        end
+  describe "#create" do
+    let(:like_hash) {
+      {positive: 1, post_id: @target.id}
+    }
+    let(:dislike_hash) {
+      {positive: 0, post_id: @target.id}
```

Comment:
I think we can just drop this `dislike` stuff (also `positive` on `like_hash`), because it is not used anywhere.